config/navigation.sample.json

```diff
@@ -0,0 +1,34 @@
+{
+  "primary": [
+    {
+      "path": "/works",
+      "text": "Works"
+    },
+    {
+      "path": "/metadata",
+      "text": "Metadata"
+    },
+    {
+      "path": "/map",
+      "text": "Map"
+    },
```

Comment:
At some point I wonder if we could just collapse some of this into something like `Explore`.

Reply:
Yes!

**Explore**
-- Works
-- Collections
-- Metadata
-- Maps
-- Timeline